source/MRMesh/MRLog.h

```diff
@@ -17,6 +24,15 @@ class MR_BIND_IGNORE Logger
 public:
     MRMESH_API static Logger& instance();
 
+    /// simple methods to log messages
+    /// for more optimal methods use getSpdLogger()
+    MRMESH_API static void trace( std::string_view msg );
+    MRMESH_API static void debug( std::string_view msg );
+    MRMESH_API static void info( std::string_view msg );
+    MRMESH_API static void warn( std::string_view msg );
+    MRMESH_API static void error( std::string_view msg );
+    MRMESH_API static void critical( std::string_view msg );
```

Comment:
i think it is not very useful since we use `spdlog::info("{} {}",asd,asd)` almost alwyas